app/src/main/java/com/example/sobok_android/presentation/view/pill/add/adapter/PillNameAdapter.kt

```diff
@@ -54,25 +62,33 @@ class PillNameAdapter(
                 binding.tvPillName.text = null
             }
 
-            binding.tvPillName.addTextChangedListener(object : TextWatcher {
-                override fun afterTextChanged(s: Editable?) {
-                }
+            binding.tvPillName.addTextChangedListener(
+                object : TextWatcher {
+                    override fun afterTextChanged(s: Editable?) {
+
+                    }
 
```

Comment:
이 부분이 afterTextChanged, beforeTextChanged에서 할 일이 없고
onTextChanged에서만 연산이 필요한 거라면 object를 만들 필요 없이 clicklistener처럼 하셔도 됩니다!
-> 0으로 초기 세팅을 원하면 textChangedListener 전에 `tvTextCount.text = "0/ 10"` 한 번 써주시면 될 듯요!

```suggestion
            with(binding) {
                   tvPillName.addTextChangedListener{
                       addName?.invoke(position, s.toString())
                       isEmpty = tvPillName.text.isEmpty()
                       tvTextCount.text = tvPillName.length().toString() + " / 10"
                  }
            }

```

Reply:
25줄을 7줄로 줄였다🥹 반영했슴다!